pkg/controller/kubelet-config/kubelet_config_nodes.go

```diff
@@ -77,6 +77,11 @@ func (ctrl *Controller) syncNodeConfigHandler(key string) error {
 		err := fmt.Errorf("could not fetch Node: %w", err)
 		return err
 	}
+	// Set the Cgroups version explicitly to "v1"
+	if nodeConfig.Spec.CgroupMode == emptyInput {
+		nodeConfig.Spec.CgroupMode = osev1.CgroupModeV1
+		ctrl.configClient.ConfigV1().Nodes().Update(context.TODO(), nodeConfig, metav1.UpdateOptions{})
```

Comment:
Hmm, is it needed to update the config back? I see below you are updating the MachineConfig entry to have the right default, would that be enough?

This also could fail due to e.g. conflict writes, transient networking issues, etc. so if we dont need this maybe we just keep the node config intact (and empty) and just change MC values. WDYT?

Reply:
Thought: As of now, we overwrite the CGroup version "V1" even if it is found to be "". This update makes sense here, but in future, if we don't need to overwrite, this update doesn't make sense (updating the same already existing Config Node object).
Hence, I would remove this step.